windows/ReactTestApp/MainPage.h

```diff
@@ -31,6 +33,9 @@ namespace winrt::ReactTestApp::implementation
             Windows::Foundation::IInspectable const &);
         void SetUpTitleBar();
     };
+
+    void WritePropertyValue(std::any propertyValue,
+                            winrt::Microsoft::ReactNative::IJSValueWriter writer);
```

Comment:
We should pass in const references to avoid copies:
```suggestion
    void WritePropertyValue(const std::any &propertyValue,
                            const winrt::Microsoft::ReactNative::IJSValueWriter &writer);
```